h2o-genmodel/src/main/java/hex/genmodel/algos/glrm/GlrmMojoModel.java

```diff
@@ -70,54 +76,52 @@ public GlrmMojoModel(String[] columns, String[][] domains, String responseColumn
     return _ncolX;
   }
 
+  public double[] setAlphas(int numAlpha) {
+    double[] alphas = new double[numAlpha];
+    double alpha = 1.0;
+    for (int index=0; index < numAlpha; index++) {
+      alpha *= DOWN_FACTOR;
+      alphas[index] = alpha;
+    }
+    return alphas;
+  }
+
   public double[] score0(double[] row, double[] preds, long seedValue) {
     assert row.length == _ncolA;
     assert preds.length == _ncolX;
     assert _nrowY == _ncolX;
     assert _archetypes.length == _nrowY;
     assert _archetypes[0].length == _ncolY;
-    double alpha=1.0;  // reset back to 1 for each row
+    _allAlphas = setAlphas(_numAlphaFactors);
 
     // Step 0: prepare the data row
     double[] a = getRowData(row);
 
     // Step 1: initialize X  (for now do Random initialization only)
```

Comment:
> for now do Random initialization only
If we trained GLRM model with PlusPlus initialization, will this matter in GLRM mojo scoring?

Reply:
For loss function that generates multiple local minima, this will matter.  It basically means you are starting from different random values.  However, during training, if you set the random seed, then, model predict and mojo predict will start from the same random seeds and you could likely fall into the same local minimum during training predict and mojo predict.

Reply:
Got it. Thanks for explaining @wendycwong